ipn/prefs.go

```diff
@@ -229,6 +233,13 @@ type AutoUpdatePrefs struct {
 	Apply bool
 }
 
+// AppConnectorPrefs are the app connector settings for the node agent.
+type AppConnectorPrefs struct {
```

Comment:
Is this stuff all diffbase from the other PR?

Reply:
Looks like it yes, I guess it doesn't always handle the rebase properly.